app/utils/get_filtered_records.py

```diff
@@ -129,7 +128,7 @@ def get_all_records(research_fields=False, include_disputed_regions=False):
 
 
 def get_filtered_records(research_fields=False, filters=None, columns=None, include_disputed_regions=False,
-                         sampling_start_date=None, sampling_end_date=None,
+                         sampling_start_date=None, sampling_end_date=None, include_subgeography_estimates=False,
```

Comment:
Is this how we want to expose this? This isn't very general. I would prefer something like:

pool = False
subgroup_estimates_by = ('subgroup_var', 'Geographical area')
include_primary_estimate_when_pooling = True

Reply:
Depends how much control we want to allow our users to have. With more control comes more confusion and more room for error. I would rather design for what we know we will need and what we know people will be able to use properly.